src/location.c

```diff
@@ -545,11 +545,11 @@ handle_start_in_thread_func (GTask *task,
           info = g_desktop_app_info_new (id);
           name = g_app_info_get_display_name (G_APP_INFO (info));
 
-          title = g_strdup_printf (_("Give %s Access to Your Location?"), name);
+          title = g_strdup_printf (_("Allow %s to Access Your Location?"), name);
```

Comment:
The latest [mockups](https://gitlab.gnome.org/Teams/Design/os-mockups/-/raw/master/portals/portals.png) have the following wording:

```
Allow %s to Access Location?
```